scipy/sparse/linalg/isolve/tests/test_lgmres.py

```diff
@@ -88,8 +87,6 @@ def test_outer_v(self):
         assert_(count_1 < count_0/2)
         assert_(allclose(x1, x0, rtol=1e-14))
 
-    @pytest.mark.skipif(python_implementation() == 'PyPy',
-                        reason="Fails on PyPy CI runs. See #9507")
```

Comment:
Can you clarify--are you planning to turn PyPy testing back on in our CI? The CI on this PR is passing, but we removed PyPy from CI quite some time ago unless I'm mistaken?

It was right after the `1.0` release point I think, because the maintenance burden with PyPy CI issues was getting a bit much for the team to handle.

Reply:
I was just cleaning up code related to #12711. I thought I had seen recent PyPy CI tests but I must be mistaken.

Pros of cleaning these up.
1. These tests seem to pass in PyPy (I tested locally)
2. Every `skipif` makes me a little sad

Con of this PR
1. At some future point we want to turn on PyPy and these fail in CI and someone spends time not realizing that they have a history.

@tylerjereddy take a gut call; If you think it's trivial and low risk let's merge. If you think this is not worth the effort close this and I'll delete the branch.

Reply:
The issue was, if I remember correctly, that the tests were 3 times passing 2 times failing and we never managed to find out why (including Pypy devs). Hence it was very inconsistent which eventually led us to remove the PyPy runs which was blocking all the PRs at some point.